src/Pmi/Application/HpoApplication.php

```diff
@@ -185,6 +185,7 @@ public function setHeaders(Response $response)
             . " 'unsafe-inline'" // for the places we are using inline JS
             . " www.google-analytics.com www.googletagmanager.com" // Google Analytics
             . " storage.googleapis.com" // for SOP PDFs stored in a Google Storage bucket
+            . " www.youtube.com" // for training videos hosted on YouTube
```

Comment:
It might have been a product of something else, but I needed to whitelist more domains for the video player on Research Hub than you have listed here:

```
                'www.youtube.com', // YouTube player
                's.ytimg.com', // YouTube player
                'static.doubleclick.net', // YouTube player
                'googleads.g.doubleclick.net', // YouTube player
                '*.googlevideo.com', // YouTube player
```

Reply:
I'm ok with blocking doubleclick.net (ad tracking).  The other domains (`s.ytimg.com` and `*.googlevideo.com`) are being requested from inside frame and are not being blocked (presumably because of the YouTube CSP?).  As best I can tell, in both Chrome and Firefox, the videos play fine with the CSP as is.